src/ui/Button.js

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
```

Comment:
I like to create UI components in a separate folder that are styled consistently. With this component both the "Search" and "Add Post" buttons are styled consistently. Other elements I'd consider styling are input tags, headers, links and more.